src/internal/operations/test/helper_test.go

```diff
@@ -625,3 +623,25 @@ func newIntegrationTesterSetup(t *testing.T) intgTesterSetup {
 
 	return its
 }
+
+func getTestExtensionFactories() []extensions.CreateItemExtensioner {
+	return []extensions.CreateItemExtensioner{
+		&extensions.MockItemExtensionFactory{},
+	}
+}
+
+func verifyExtensionData(
+	t *testing.T,
+	itemInfo details.ItemInfo,
+	p path.ServiceType,
+) {
+	assert.NotNil(t, itemInfo.Extension, "nil extension")
```

Comment:
require here will save you a nil panic if the extension is nil.
```suggestion
	require.NotNil(t, itemInfo.Extension, "nil extension")
```